internal/packager/init.go

```diff
@@ -79,22 +79,51 @@ func checkComposeFileVersion(compose map[string]interface{}) error {
 	return schema.Validate(compose, fmt.Sprintf("%v", version))
 }
 
-func initFromComposeFile(name string, composeFile string) error {
-	logrus.Debugf("Initializing from compose file %s", composeFile)
-
-	dirName := internal.DirNameFromAppName(name)
-
-	composeRaw, err := ioutil.ReadFile(composeFile)
-	if err != nil {
-		return errors.Wrap(err, "failed to read compose file")
+func getEnvFiles(svcName string, envFileEntry interface{}) ([]string, error) {
+	var envFiles []string
+	switch envFileEntry.(type) {
+	case string:
+		envFiles = append(envFiles, envFileEntry.(string))
+	case []interface{}:
+		for _, env := range envFileEntry.([]interface{}) {
+			envFiles = append(envFiles, env.(string))
+		}
+	default:
+		return nil, fmt.Errorf("unknown entries in 'env_file' for service %s -> %v",
+			svcName, envFileEntry)
 	}
-	cfgMap, err := composeloader.ParseYAML(composeRaw)
-	if err != nil {
-		return errors.Wrap(err, "failed to parse compose file")
+	return envFiles, nil
+}
+
+func checkEnvFiles(cfgMap map[string]interface{}) error {
+	services := cfgMap["services"]
+	servicesMap, ok := services.(map[string]interface{})
+	if !ok {
+		return fmt.Errorf("could not process the yaml file")
 	}
-	if err := checkComposeFileVersion(cfgMap); err != nil {
-		return err
+	for svcName, svc := range servicesMap {
+		svcContent, ok := svc.(map[string]interface{})
+		if !ok {
+			return fmt.Errorf("could not process the yaml file")
+		}
+		envFileEntry, ok := svcContent["env_file"]
+		if !ok {
+			continue
+		}
+		envFiles, err := getEnvFiles(svcName, envFileEntry)
+		if err != nil {
+			return errors.Wrap(err, "could not process the yaml file")
+		}
+		for _, envFilePath := range envFiles {
+			fmt.Printf("\"env_file: %s\" entry of service %q will NOT be copied automatically! "+
```

Comment:
`"%s.env_file %q will not be copied to the docker application. Please copy it manually and udpate the path accordingly in the compose file.", svcName, envFilePath` ?

Reply:
^ @chris-crone

Reply:
I would say:
```"%s.env_file %q will not be copied into %s.dockerapp. Please copy it manually and udpate the path accordingly in the compose file.", svcName, appName, envFilePath```